.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image:
+  file: .docker/gitpod/mathlib/Dockerfile
+
+vscode:
+  extensions:
+    - jroesch.lean@0.16.24:wYErjZXBf8vkagq/BhUHJA==
+
+tasks:
+  - command: . /home/gitpod/.profile && leanproject get-mathlib-cache
```

Comment:
Why do we need to get mathlib twice? (Once in the docker image, and then on startup here.)

Reply:
Yes, this is stupid. The reason I did it was that I wanted to make sure that the latest version of the lean toolchain was baked into the docker image. It seems that even though `elan-init` downloads a toolchain, the first time you run `leanproject ...` in mathlib it pulls down another one.

Help on this definitely welcome!

Reply:
I've fixed this! This saves 290mb off the container, so it's now only 115mb of layers beyond `gitpod/workspace-base`. Hopefully this improves load times.

Reply:
Still seems pretty slow to me.. About 12s from click to VSCode popping up. :-(